ViewLayer/DebugDisplay/MotorFaultPage/MotorFaultView/MotorFaultView.cpp

```diff
@@ -0,0 +1,2819 @@
+#include "MotorFaultView.h"
+
+namespace
+{
+    int height = 25;
+    QString errorStyleSheet = "font: 20px 'Arial';\nfont-weight:500;color:red; margin-left: 10px;";
+    QString limitStyleSheet = "font: 20px 'Arial';\nfont-weight:500;color:yellow; margin-left: 10px;";
+    int labelResizeLimit = 5;
+}
+MotorFaultView::MotorFaultView(MotorFaultsPresenter& motorFaultsPresenter,
+                               BatteryFaultsPresenter& batteryFaultsPresenter,
+                               I_MotorFaultUi& ui)
+    : motorFaultsPresenter_(motorFaultsPresenter)
+    , batteryFaultsPresenter_(batteryFaultsPresenter)
+    , ui_(ui)
+    , badMotorPositionHallSequence0Fault_(new QLabel())
```

Comment:
For these QLabels, we can initialize it with the string we want the label to have.
`badMotorPositionHallSequence0Fault_("Bad Motor Position Hall Sequence Error");`